sdk/agrifood/azure-agrifood-farming/samples/async/sample_hello_world_async.py

```diff
@@ -0,0 +1,67 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+from typing import Dict
+from azure.identity.aio import DefaultAzureCredential
+from azure.agrifood.farming.aio import FarmBeatsClient
+from azure.agrifood.farming.models import Farmer
+import asyncio
+import platform
+import os
+
+
+async def main(client: FarmBeatsClient, config: Dict):
+
+    print("Creating farmer, or updating if farmer already exists...", end=" ", flush=True)
+    farmer = await client.farmers.create_or_update(
+        farmer_id=config['farmer_id'],
+        farmer=Farmer(
+            name=config['farmer_name'],
+            description=config['farmer_description']
+        )
+    )
+    print("Done")
+    print("Here are the details of the farmer:")
+    print(f"ID: {farmer.id}")
+    print(f"Name: {farmer.name}")
+    print(f"Description: {farmer.description}")
+    print(f"Created timestamp: {farmer.created_date_time}")
+    print(f"Last modified timestamp: {farmer.modified_date_time}")
+
+
+if __name__ == "__main__":
+
+    # config contains any data that might be used while running the sample.
+    config = {
+        'farmer_id': "contoso-farmer",
+        'farmer_name': "Contoso",
+        'farmer_description': "Contoso is hard working."
+    }
+
+    try:
+        farmbeats_endpoint = os.environ.get('FARMBEATS_ENDPOINT')
+    except KeyError:
+        farmbeats_endpoint = None
+
+    if farmbeats_endpoint is None:
+        raise SystemExit("Please set the 'FARMBEATS_ENDPOINT' env variable to your FarmBeats endpoint.")
```

Comment:
instead of doing this error checks, you can just do `os.environ['FARMBEATS_ENDPOINT']`. If that fails, it's more clear to users that they just need toset that envar. See example [here](https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/textanalytics/azure-ai-textanalytics/samples/sample_analyze_actions.py#L44)

Reply:
![image](https://user-images.githubusercontent.com/23560829/123006775-d3cf3900-d3d5-11eb-9032-975463335651.png)
This doesn't explicitly say **what to do** to fix. Wanted to reduce funnels as much as possible. Thoughts?

Reply:
Discussed with folks on my side, we wanna keep this :) Cleaned it up a little bit though.